packages/@okta/vuepress-site/docs/guides/quickstart/create-org/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Create your Okta organization
+---
+
+You built an awesome app and you want to add user authentication to it. Let’s get you set up and working with Okta.
+
+To start, you're going to need a free Okta developer edition org. An org is a private data space Okta creates for you, to hold all the resources you create to handle user authentication.
+
+An org is free, and you can use it to handle authentication for up to 1,000 users. Later, if you need more capacity, you can upgrade to a paid org.
+
+#### Create your org
+
+<StackSelector snippet="create-org" />
```

Comment:
Let's make sure to add human-formatted strings for `CLI` and `Website` in [util/frameworks.js](https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-theme-prose/util/frameworks.js).